frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behavior-tab.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-behavior-tab',
```

Comment:
Please make the selector more specifiy. Currently it does not convey that it belongs to a simulated region popup